Slider/Assets/Scripts/Systems/Save/SaveProfile.cs

```diff
@@ -130,15 +139,19 @@ public void Save()
         //Debug.Log(playTimeInSeconds);
         SetBool("isDemoBuild", true);
         SaveSavablesData();
+        AchievementData = AchievementManager.GetAchievementData();
```

Comment:
should we save achievement data separately since they are global, whereas these save profiles are all individually saved? might make sense to put them in player prefs maybe (altho its kinda gross)